src/extras/core/Path.js

```diff
@@ -168,7 +168,13 @@ THREE.Path.prototype.ellipse = function ( aX, aY, xRadius, yRadius,
 THREE.Path.prototype.absellipse = function ( aX, aY, xRadius, yRadius,
 									  aStartAngle, aEndAngle, aClockwise, aRotation ) {
 
-	var args = Array.prototype.slice.call( arguments );
+	var args = [
+		aX, aY,
+		xRadius, yRadius,
+		aStartAngle, aEndAngle,
+		aClockwise,
+		aRotation || 0 // aRotation is optional.
+	];
```

Comment:
@neko1235 Yes, you can comment on closed PRs... If you think you can improve the design, please have a go, and file a PR. Also, there is #6972. If you'd be willing to propose a fix, that would be time-well-spent.